src/wasm-s-parser.h

```diff
@@ -160,6 +160,14 @@ class SExpressionWasmBuilder {
   }
   Type
   stringToType(const char* str, bool allowError = false, bool prefix = false);
+  HeapType stringToHeapType(cashew::IString str,
+                            bool allowError = false,
+                            bool prefix = false) {
+    return stringToHeapType(str.str, allowError, prefix);
+  }
+  HeapType stringToHeapType(const char* str,
+                            bool allowError = false,
+                            bool prefix = false);
```

Comment:
Can these be deferred to a later PR?

Reply:
I guess we need these now given that we want to parse and print `(ref.null extern)` for example.

Reply:
Makes sense, thanks 👍